doc/CustomReorgAppletExample.md

```diff
@@ -73,7 +73,15 @@ def main(reorg_conf___=None, reorg_status___=None):
 
     # find the output stage of the current analysis
     analysis_id = dxpy.describe(dxpy.JOB_ID)["analysis"]
-    stages = dxpy.describe(analysis_id)["stages"]
+    # describe the analysis in a loop until dependsOn is empty
+    while True:
+        analysis_desc = dxpy.describe(analysis_id)
+        if analysis_desc.get("dependsOn"):
+            time.sleep(1)
```

Comment:
Is one second enough to wait? It usually takes more than 1s to describe. Won't the user eventually get throttled?